tools/acrn-manager/acrnctl.c

```diff
@@ -193,6 +193,21 @@ static int acrnctl_do_add(int argc, char *argv[])
 			printf("Too many optional args: %s\n", args);
 			return -1;
 		}
+
+		/*
+		 * If there's "-C" parameter in acrnctl add command
+		 * check if the SoS support runC container at first, then
+		 * strip "-C" and set the flag.
+		*/
+		if (strncmp(argv[i], "-C", 2) == 0) {
+			if (access("/sbin/runc", F_OK) != 0) {
```

Comment:
I'm concerned that this would easily break on systems that do have `runc` installed but possibly on a different path (e.g., I believe Ubuntu ships `runc` in `/usr/sbin`).

Reply:
This message only appears when the user use acrnctl add command with "-C" parameter.